frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -657,6 +661,35 @@ export const surveyLogic = kea<surveyLogicType>([
                     const newTemplateSurvey = { ...NEW_SURVEY, ...template }
                     return newTemplateSurvey
                 },
+                setQuestionBranching: (state, { questionIndex, value }) => {
+                    const newQuestions = [...state.questions]
+                    const question = newQuestions[questionIndex] as RatingSurveyQuestion | MultipleSurveyQuestion
```

Comment:
hmm, instead of type coercion here, I'd much prefer a check which returns early if the question isn't a rating or multiple choice question. This prevents borking if this `setQuestionBranching` function is called incorrectly.